### PR TITLE
Add send log events payload too big 2 data sender imple test test max payload test

### PR DESCRIPTION
**Pull request by twcrone**
Add tests for send log events with too big of a payload

### Related Github Issue
https://github.com/newrelic/newrelic-java-agent/issues/735

### Checks

[Y] Are your contributions backwards compatible with relevant frameworks and APIs?
[N] Does your code contain any breaking changes? Please describe. 
[N] Does your code introduce any new dependencies? Please describe.

### CODE REVIEW

File: newrelic-agent/src/test/java/com/newrelic/agent/transport/DataSenderImplTest.java [hunk 1]
```diff
@@ -50,10 +51,7 @@
 import java.net.URL;
 import java.security.NoSuchAlgorithmException;
 import java.text.MessageFormat;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```
Comment (jasonjkeller) [inline newrelic-agent/src/test/java/com/newrelic/agent/transport/DataSenderImplTest.java:54]: Make sure that you're using the java-agent code style found at `newrelic-java-agent/dev-tools/code-style/java-agent-code-style.xml`. It's based on the [Google Style Guide](https://google.github.io/styleguide/javaguide.html) which doesn't allow wildcard imports. You can import the scheme in **Preferences > Editor > Code Style > Java > Scheme**.

**Review by jasonjkeller — APPROVED** [review]
One small formatting tweak otherwise looks good to 🐐 .

File: newrelic-agent/src/test/java/com/newrelic/agent/transport/DataSenderImplTest.java (hunk 1 above)
Comment (twcrone) [reply newrelic-agent/src/test/java/com/newrelic/agent/transport/DataSenderImplTest.java]: Done

**Review by tbradellis — APPROVED** [review]
I approve this message.

**Pull request merged**